services-js/commissions-app/src/client/ApplicantInformationSection.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { Field } from 'formik';
+import { css } from 'emotion';
+
+import { MEDIA_MEDIUM } from '@cityofboston/react-fleet';
+
+import TextInputContainer from './common/TextInputContainer';
+
+
+const COMMON_MARGIN = 0.75;
+
```

Comment:
These CSS chunks could probably use an overall comment explaining the strategy. You might also want a name a bit more specific than `COMMON`. Does `ROW_STYLING` fit?

Reply:
You should include a note about why you went w/ this and not the grid classes.